test/us03_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import sys
+import os
+sys.path.append(os.path.abspath('../src'))
+os.chdir(os.path.abspath('../test/seeds'))
+import us03
+
+class TestUser03(unittest.TestCase):
```

Comment:
nit: function name should match file name

Reply:
fixed